Fix Gloo Edge install when failurePolicy is Fail (#6349)
solo-io/gloo: docs/content/reference/values.txt

|gateway.certGenJob.cron.enabled|bool|false|enable the cronjob|
|gateway.certGenJob.cron.schedule|string|* * * * *|Cron job scheduling|
|gateway.certGenJob.cron.mtlsKubeResourceOverride.NAME|interface||override fields in the gloo-mtls-certgen cronjob.|

> |gateway.rolloutJob.image.tag|string|1.22.9|tag for the container|

Reviewer: ```suggestion
|gateway.rolloutJob.image.tag|string|1.22.9| The image tag for the container.|
```

Author: all of these descriptions on the Jobs were inherited from existing structs. i can fix these but we'll probably need to do another pass to make all the descriptions throughout the file consistent (the source file is `install/helm/gloo/generate/values.go` which generates this values.txt)

|gateway.rolloutJob.image.repository|string|kubectl|image name (repository) for the container.|

> |gateway.rolloutJob.image.registry|string|bitnami|image prefix/registry e.g. (quay.io/solo-io)|

Reviewer: ```suggestion
|gateway.rolloutJob.image.registry|string|bitnami|The image hostname prefix and registry, such as quay.io/solo-io.|
```

> |gateway.rolloutJob.image.pullPolicy|string||image pull policy for the container|

Reviewer: Is there a default set? Maybe it varies, based on the Kubernetes default?
```suggestion
|gateway.rolloutJob.image.pullPolicy|string||The image pull policy for the container. For default values, see the Kubernetes docs: https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting|
```

> |gateway.rolloutJob.image.pullSecret|string||image pull policy for the container |

Reviewer: Should this be secret? Is there a default or any other caveat that might be helpful to include?
```suggestion
|gateway.rolloutJob.image.pullSecret|string||The image pull secret to use for the container, in the same namespace as the container pod.|
```

|gateway.rolloutJob.image.extended|bool||if true, deploy an extended version of the container with additional debug tools|

> |gateway.rolloutJob.image.fips|bool||if true, deploy a version of the container (for data-plane containers) built with FIPS-compliant crypto libraries (enterprise only)|

Reviewer: ```suggestion
|gateway.rolloutJob.image.fips|bool||If true, deploys a version of the data-plane containers that is built with FIPS-compliant crypto libraries. (Enterprise-only feature.)|
```

> |gateway.rolloutJob.floatingUserId|bool||set to true to allow the cluster to dynamically assign a user ID|

Reviewer: What is the user ID assigned to?
```suggestion
|gateway.rolloutJob.floatingUserId|bool||If true, allows the cluster to dynamically assign a user ID for the container.|
```
